src/main/java/mod/chiselsandbits/core/api/ChiselAndBitsAPI.java

```diff
@@ -322,4 +330,22 @@ public void endUndoGroup(
 		UndoTracker.getInstance().endGroup( player );
 	}
 
+	@Override
+	@SideOnly( Side.CLIENT )
+	public KeyBinding getKeyBinding(
+			ModKeyBinding modKeyBinding )
+	{
+		switch ( modKeyBinding.type )
```

Comment:
True... It would probably be better to remove the nested type enum in ModKeyBinding and just replace this switch on the type with one big switch on all the values not in ClientSide.